### PR TITLE
Basic work on fixing up logging

### PR DESCRIPTION
**Pull request by danquah**
Main work done, the search-node and middleware is remaining.

REYOS2-5

### CODE REVIEW

File: images/_versions.source [hunk 1]
```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env bash
 
 KKOS_IMAGE_REPOSITORY=kkos2display
+#MAIN_BASE_IMAGE=gcr.io/gcp-runtimes/ubuntu_18_0_4:latest
```
Comment (tripox) [inline images/_versions.source]: Huh?

**Comment by naxoc** [comment]
I'v been building this for developemnt and it takes _FOREVER_. It seems it's a desktop Ubuntu being pulled in - complete with icon packs and everything. By _FOREVER_ I mean over 20 mins. The build was much faster before.

When I run `make up` I get this:
```
ERROR: for admin_release  Cannot start service admin_release: OCI runtime create failed: container_linux.go:344: starting container process caused "exec: \"/opt/init-copy-source.sh\": stat /opt/init-copy-source.sh: no such file or directory": unknown
ERROR: Encountered errors while bringing up the project.
Makefile:18: recipe for target 'up' failed
make: *** [up] Error 1
```
Am I missing something?

**Comment by tripox** [comment]
`init-copy-source.sh` apparently, @naxoc :D

**Comment by naxoc** [comment]
Yes, that file is missing. It's there in the checkout - it just doesn't get mounted into where it should go.

**Comment by naxoc** [comment]
I'm merging - master is missing the gosu stuff :)

**Pull request merged**